torch_brain/data/dataset.py

```diff
@@ -286,6 +286,7 @@ def get(self, recording_id: str, start: float, end: float):
         sample.units.id = np.core.defchararray.add(
             f"{sample.brainset}/{sample.session}/", sample.units.id.astype(str)
```

Comment:
Since you are at it, I would also fix this error:
```suggestion
            f"{sample.brainset.id}/{sample.session.id}/", sample.units.id.astype(str)
```